optional-requirements.txt

```diff
@@ -1,4 +1,4 @@
 # Optional Storage
-python-rocksdb
+#python-rocksdb
```

Comment:
Shouldn't we keep this in the requirements in case the user wants to use `RocksDB`?

Reply:
I had difficulty installing `python-rocksdb` on my laptop and also on Travis. If a user wants to use RocksDB, he/she will have find a way to install and get it working. So I have just commented out this optional requirement, instead of removing it completely.

Reply:
Maybe leave a comment about why it is commented out?